app/Http/Controllers/VoteController.php

```diff
@@ -45,13 +47,18 @@ private function getMetaData($votes)
         $data = [];
 
         foreach ($votes as $vote) {
-
+            $usersWhoSaw = [];
+            foreach ($vote->voteUniqueViews()->get()->load('user') as $view) {
+                $usersWhoSaw[] = $view->user;
```

Comment:
I think that users list might be get another way. You can retrieve all related users by relation